mistralrs-core/src/pipeline/gguf.rs

```diff
@@ -353,10 +353,11 @@ impl Loader for GGUFLoader {
                 self.get_id(),
                 device.device_pretty_repr()
             );
-        } else if paged_attn_config.is_some() {
-            warn!("Device mapping or device topology and PagedAttention are incompatible, disabling PagedAttention.");
-            paged_attn_config = None;
         }
+        // } else if paged_attn_config.is_some() {
```

Comment:
Is there a reason why this was commented out instead of removed? It seems that the change adds full support.